Framework/Muon/src/MuonAlgorithmHelper.cpp

```diff
@@ -516,5 +516,86 @@ bool checkValidGroupPairName(const std::string &name) {
   return true;
```

Comment:
this file feels a bit like a dumping ground to me. I think we could split this up into more sensible helpers (e.g. naming, runNumber, periods). I suspect it may be a big job so should be done in another issue.